Packs/AutoFocus/Integrations/AutoFocusTagsFeed/AutoFocusTagsFeed.py

```diff
@@ -0,0 +1,509 @@
+         ThreatIntel.ObjectsNames.ATTACK_PATTERN: 'used-by'},
+    ThreatIntel.ObjectsNames.ATTACK_PATTERN:
+        {ThreatIntel.ObjectsNames.MALWARE: 'uses',
+         ThreatIntel.ObjectsNames.THREAT_ACTOR: 'used-by',
+         ThreatIntel.ObjectsNames.CAMPAIGN: 'uses',
+         ThreatIntel.ObjectsNames.ATTACK_PATTERN: 'related-to'},
+
+}
+
+SCORES_MAP = {
+    ThreatIntel.ObjectsNames.MALWARE: ThreatIntel.ObjectsScore.MALWARE,
+    ThreatIntel.ObjectsNames.THREAT_ACTOR: ThreatIntel.ObjectsScore.THREAT_ACTOR,
+    ThreatIntel.ObjectsNames.CAMPAIGN: ThreatIntel.ObjectsScore.CAMPAIGN,
+    ThreatIntel.ObjectsNames.ATTACK_PATTERN: ThreatIntel.ObjectsScore.ATTACK_PATTERN,
+}
+
+# The page size in the AutoFocus response (can be 1-200)
+PAGE_SIZE = 50
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """
+    Client class to interact with AutoFocus API
+    Args:
+        api_key: AutoFocus API Key.
+    """
+
+    def __init__(self, api_key, verify, proxy):
+        super().__init__(BASE_URL, verify, proxy)
+        self.api_key = api_key
+
+    def get_tags(self, data: Dict[str, Any]):
+        res = self._http_request('POST',
+                                 url_suffix='tags',
+                                 headers={
+                                     'apiKey': self.api_key,
+                                     'Content-Type': 'application/json'
+                                 },
+                                 json_data=data,
+                                 timeout=60,
+                                 )
+        return res
+
+    def get_tag_details(self, public_tag_name: str):
+        res = self._http_request('POST',
+                                 url_suffix=f'tag/{public_tag_name}',
+                                 headers={
+                                     'apiKey': self.api_key,
+                                     'Content-Type': 'application/json'
+                                 },
+                                 timeout=60,
+                                 )
+        return res
+
+    def build_iterator(self, is_get_command: bool, limit: int = -1) -> list:
+        """
+        Retrieves all entries from the feed.
+        This method implements all of the logic to get the tags from the feed.
+        Args:
+            limit:
+            is_get_command: is this method called from the get-indicators-command
+        Returns:
+            A list of objects, containing the indicators.
+        """
+
+        results = []
+        if is_get_command:
+            page_num = 0
+        else:
+            integration_context = get_integration_context()
+            if not integration_context:
+                page_num = 0
+                time_of_first_fetch = date_to_timestamp(datetime.now(), DATE_FORMAT)
+                set_integration_context({'time_of_first_fetch': time_of_first_fetch})
+            else:
+                page_num = arg_to_number(integration_context.get('page_num', 0)) or 0
```

Comment:
why do you need to convert it to int? aren't you saving it to the context as a number?